multistream-select/src/dialer_select.rs

```diff
@@ -95,6 +95,7 @@ pub fn dialer_select_proto_serial<'a, R, I, P>(
                     })
                     // Once read, analyze the response.
                     .and_then(|(message, rest, proto_name, proto_value)| {
+						if message.is_none() { println!("empty"); }
```

Comment:
I know these `println!s` are mine, but it would be nice to remove them.

Reply:
I thought I'd got rid of this, it must have been as part of the commit I rolled back